### PR TITLE
Icons doc

### PR DESCRIPTION
**Pull request by wenfeng-xing**
Screen Shot 2022-10-28 at 11 44 03 AM
Screen Shot 2022-10-28 at 11 44 57 AM

### CODE REVIEW

File: docs/.storybook/preview-body.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <style>
   body #root {
-    width: 100vw;
-    height: 100vh;
+    min-width: 100vw;
```
Comment (wenfeng-xing) [inline docs/.storybook/preview-body.html:3]: adapt the screen size to different documentation pages.

File: docs/src/design-system/typography/types-page.tsx [hunk 2]
```diff
@@ -2,10 +2,8 @@ import { Divider, Grid, Typography, useTheme } from '@mui/material';
 import { MERRIWEATHER, ROBOTO } from '@ui-kit-2022/theme';
 import * as React from 'react';
 
-import FontExamples from '../../theme/typography/examples/font-examples';
-import TextSample from '../../theme/typography/examples/text-sample';
-import TypeCard from '../../theme/typography/examples/type-card';
 import TopBar from '../building-blocks/common/top-bar';
+import { FontExamples, TextSample, TypeCard } from '../building-blocks/typography';
```
Comment (wenfeng-xing) [inline docs/src/design-system/typography/types-page.tsx:6]: An import error in the current code base

File: packages/ui-kit-components/src/icons/index.ts [hunk 3]
```diff
@@ -13,16 +13,21 @@ export { ReactComponent as ChevronUp } from './images/ChevronUpIcon.svg';
...
 export { ReactComponent as Cog } from './images/CogIcon.svg';
 export { ReactComponent as Dots } from './images/DotsIcon.svg';
+export { ReactComponent as Download } from './images/DownloadIcon.svg';
 export { ReactComponent as Edit } from './images/EditIcon.svg';
+export { ReactComponent as Eye } from './images/EyeIcon.svg';
 export { ReactComponent as Filter } from './images/FilterIcon.svg';
 export { ReactComponent as Groups } from './images/GroupsIcon.svg';
 export { ReactComponent as Info } from './images/InfoIcon.svg';
 export { ReactComponent as Lock } from './images/LockIcon.svg';
+export { ReactComponent as More } from './images/MoreIcon.svg';
```
Comment (iandilling-adaptive) [inline packages/ui-kit-components/src/icons/index.ts:23]: I think the `More` icon is a duplicate of `Dots` and the `Eye` icon is a duplicate of `View`.

**Review by iandilling-adaptive — CHANGES_REQUESTED** [review]

File: packages/ui-kit-components/src/icons/index.ts (hunk 3 above)
Comment (wenfeng-xing) [reply packages/ui-kit-components/src/icons/index.ts]: They have been replaced by view and dots icons.

**Review by iandilling-adaptive — APPROVED** [review]

**Pull request merged**